bindings/java/src/main/java/org/ucx/jucx/ucp/UcpParams.java

```diff
@@ -0,0 +1,193 @@
+/*
+ * Copyright (C) Mellanox Technologies Ltd. 2019. ALL RIGHTS RESERVED.
+ * See file LICENSE for terms.
+ */
+
+package org.ucx.jucx.ucp;
+
+import static org.ucx.jucx.ucs.UcsBits.UCS_BIT;
+
+/**
+ * Tuning parameters for UCP library.
+ *
+ * The structure defines the parameters that are used for
+ * UCP library tuning during UCP library {@link UcpContext} "initialization".
+ *
+ * <p>UCP library implementation uses the {@link UcpParams.UcpFeature} "features"
+ * parameter to optimize the library functionality that minimize memory
+ * footprint. For example, if the application does not require send/receive
+ * semantics UCP library may avoid allocation of expensive resources associated with
+ * send/receive queues.
+ */
+public class UcpParams {
+
+    @Override
+    public String toString() {
+        return "UcpParams{"
+            + "fieldMask=" + fieldMask
```

Comment:
minor: we probably should not print fieldMask, but use it to decide which other params to print

Reply:
This is not printed anywhere for now. For debugging issue users may decide to print to compare with ucx log output:
```
 ucp_context.c:1242 UCX  DEBUG created ucp context 0x7f88d457f9a0 [4 mds 5 tls] features 0x3
 tl bitmap 0x1f
```

Reply:
fieldMask is not the same as "features"..

Reply:
Removed toString for now.